### ISSUE TITLE
Catalog GAP species literature sources for reference

### ISSUE THREAD

**Issue by skybristol** [usgs-bcb/bcb-dm#15]
There is a wealth of literature behind the GAP species models that provide the basis for species traits related to habitat affinity characteristics. Those are in some form of backend database that is being used to build the individual species model reports for the release. I'd like to get these cataloged individually as citation records (probably in ScienceBase, though we might think about other options) as a resource to build on. If there is any other attribution in the GAP database on the significance of any individual reference to the species, it would be good to look at how that might fit into a linked data structure.

In addition to being a cleaner and fully online process for managing the references, this dataset linking species to publications can be used to drive R&D into supervised literature mining processes that we are developing on the DeepDive algorithms with the dam removal use case. We can use as much of the GAP species lit reference as we can get full text for to run training processes for future automated methods.

**Comment by nmtarr** [comment]
I agree that the literature behind the models is a valuable resource that people may want access to.  The references for the habitat models are currently stored in a table within the WHRdb.  I just looked and there are 16,417 individual sources.  I think that list was created in part by combining tables from regional projects almost 10 years ago; therefore, the formats of the unique codes and reference text are somewhat inconsistent.  At the least, it seems like a human would have to review the codes and text for each of the 16,417 records after code parsed the strings to whatever new format would be used in ScienceBase.  At the most, someone would have to review and revise each one.   Not to mention, if records are incomplete, that person would have to track down the source (assuming they can get access to it) and complete it.  I think such an effort would take a very, very long time and would surely displace effort for other tasks.

**Comment by skybristol** [comment]
Thanks, @nmtarr, for the additional info. That is, indeed, a sizable resource of reference material. I'd like to take a look at the data and see what we might do with them. For the Biogeographic Information System, I think we're going to want to look something other than ScienceBase for this. One way or another, we're going to get varying degrees of current and future value from this citation list. I'd bet that the vast majority of the citation strings do not have something convenient like a DOI that we can use to go track down the actual text easily. While we can write some code to do some of that, it's never going to be 100%.

However, I'd argue for simply putting the data into an online database following a basic annotation model (https://www.w3.org/TR/annotation-model/) for the strings containing "codes and reference text." Part of that would be assigning our own (or leveraging some you already have) unique identifiers for the references to tie them to other things in the BIS like GAP species. As we have time or need, we can build out the resource through supervised code to parse annotations into more usable properties.

This would require minimal effort at the outset, would serve the purpose of daylighting the data, would let us exercise some stuff we need to do in the BIS for other purposes (e.g., we need a new model for annotated references on the "Dam Removal Database" right now as well), and would give us a foundation for future development.
